collectd.conf.tmpl

```diff
@@ -1,4 +1,5 @@
 Interval 60
+FQDNLookup false
```

Comment:
Might want to add a comment that the FQDN is not used by the Stackdriver monitoring agent.

Reply:
PTAL.